libbeat/docs/outputconfig.asciidoc

```diff
@@ -13,6 +13,12 @@
 [[configuring-output]]
 == Configure the output
 
+ifdef::only-elasticsearch[]
+You configure {beatname_uc} to write to Elasticsearch by setting options in
+the `output.elasticsearch` of the +{beatname_lc}.yml+ config file.
```

Comment:
I'd say either "in the `output.elasticsearch` section of the +{beatname_lc}.yml+ config file" or "under `output.elasticsearch` in the +{beatname_lc}.yml+ config file."